src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/ResolveSpecificCommentAuthorizationHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Linq;
```

Comment:
Oh another thing - make sure to add `#nullable enable` to files you're touching (unless there is a good reason not to e.g. too many warnings or it's a database entity)